data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report4_dateshift.py

```diff
@@ -439,16 +476,61 @@
 JOIN df6 USING(n_row_not_pass)
 JOIN df7 USING(n_row_not_pass)
 JOIN df8 USING(n_row_not_pass)
+JOIN df9 USING(n_row_not_pass)
 
 
 
 '''
 df1=execute(client, query)
 if df1.eq(0).any().any():
- df = df.append({'query' : 'Query14.3 person_id replaed by research_id in other 8 tables', 'result' : 'PASS'},  
+ df = df.append({'query' : 'Query15.3 person_id replaed by research_id in other 9 tables', 'result' : 'PASS'},  
+                ignore_index = True) 
+else:
+ df = df.append({'query' : 'Query15.3 person_id replaed by research_id in other 9 tables', 'result' : ''},  
+                ignore_index = True) 
+df1
+
+# # 16 Verify that  questionnaire_response_id/survey_conduct_id has been replaced by research_response_id
+
+query = f'''
+WITH df1 AS (
+SELECT COUNT (*) AS n_row_not_pass
```

Comment:
This comment is a "nice to have".  A SQL comment describing what each CTE is accomplishing is helpful when debugging or making maintenance changes.